index.html

```diff
@@ -63,6 +63,30 @@
 						"title": "Digital Publishing WAI-ARIA Module",
 						"href": "https://www.w3.org/TR/dpub-aria/",
 						"publisher": "W3C"
+					},
+					"css-page": {
+						"aliasOf": "css-page-3-20230914"
```

Comment:
It's best not to alias dated drafts like this as we'll be stuck with the dated version as they get updated. Is there a reason you can't use "css-page-3", for example, as those references always point to the latest published version.